### PR TITLE
Added requirements.txt & updated README

### LINKED ISSUE #4: Error running c2q_transfer_learning_ants_bees.ipynb w/ GPU
I'm trying to run the c2q_transfer_learning_ants_bees.ipynb notebook on a GPU. However, I keep encountering errors when running this line:

`model_hybrid = train_model(model_hybrid, criterion, optimizer_hybrid,exp_lr_scheduler, num_epochs=num_epochs)`

I've encountered the same error on my local machine, and Google Colab (both testing on a GPU). The code runs completely when run on a CPU.

Here's the traceback:
```
RuntimeError                              Traceback (most recent call last)
<ipython-input-43-efc42279baeb> in <module>()
----> 1 model_hybrid = train_model(model_hybrid, criterion, optimizer_hybrid,exp_lr_scheduler, num_epochs=num_epochs)

2 frames
<ipython-input-42-9f2a4a230553> in train_model(model, criterion, optimizer, scheduler, num_epochs)
     38                         loss = criterion(outputs, labels)
     39                         if phase == 'train':
---> 40                             loss.backward()
     41                             optimizer.step()
     42 

/usr/local/lib/python3.6/dist-packages/torch/tensor.py in backward(self, gradient, retain_graph, create_graph)
    183                 products. Defaults to ``False``.
    184         """
--> 185         torch.autograd.backward(self, gradient, retain_graph, create_graph)
    186 
    187     def register_hook(self, hook):

/usr/local/lib/python3.6/dist-packages/torch/autograd/__init__.py in backward(tensors, grad_tensors, retain_graph, create_graph, grad_variables)
    125     Variable._execution_engine.run_backward(
    126         tensors, grad_tensors, retain_graph, create_graph,
--> 127         allow_unreachable=True)  # allow_unreachable flag
    128 
    129 

RuntimeError: CUDA error: CUBLAS_STATUS_EXECUTION_FAILED when calling 'cublasDgemm( handle, opa, opb, m, n, k, &alpha, a, lda, b, ldb, &beta, c, ldc)'
```

### PR DESCRIPTION
**Pull request by Jerry2001Qu**
Training these notebooks on a CUDA device returned errors. Fixed when reverting package versions to when the paper/repo was originally released. Resolves #4 

- Added requirements.txt w/ reverted package versions
- Added a note in README about running on CUDA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,8 +53,9 @@ The notebooks `c2q_transfer_learning_*.ipynb` require the Python libraries Penny
 
 The notebooks `q2c_transfer_learning.ipynb` and  `q2q_transfer_learning.ipynb` require the library Strawberry Fields with the TensorFlow backend.
 
-For the requirements of the code in `quantum_processors\` see the dedicated README file `quantum_processors\README.md`.
+All notebooks require the package versions specified in the `requirements.txt` to run on a CUDA device.
```
Comment (trbromley) [inline README.md:56]:
```suggestion
Notebooks can be run using the package versions specified in the `requirements.txt` file.
```
Perhaps this way could present a nice "fall-back" for anyone experiencing compatibility issues, regardless of using CUDA or not.

**Review by trbromley — APPROVED** [review]
Thanks @Jerry2001Qu! I left a small suggestion for the wording.

Let me know when ready to merge in!

:rocket:

**Comment by Jerry2001Qu** [comment]
Awesome! Looks good to me.

**Comment by trbromley** [comment]
Thanks!

**Pull request merged**